awscli/botocore/httpsession.py

```diff
@@ -334,9 +375,7 @@ def _get_request_target(self, url, proxy_url):
             return self._path_url(url)
 
     def _chunked(self, headers):
-        transfer_encoding = headers.get('Transfer-Encoding', b'')
-        transfer_encoding = ensure_bytes(transfer_encoding)
-        return transfer_encoding.lower() == b'chunked'
+        return headers.get('Transfer-Encoding', '') == 'chunked'
```

Comment:
What prompted this change, it doesn't like this exists upstream in Botocore 1.x? Are we certain this isn't going to be breaking for any headers that are byte-encoded because of a different codec?

Reply:
Hmm. I was comparing against the v2 branch of `botocore` and took it from there, maybe I misunderstood how the vendored copy of `botocore` is there now!

https://github.com/boto/botocore/blob/v2/botocore/httpsession.py#L324-L325

Reply:
I'll undo it since it's also not in the vended copy.